ida/dbc_naming-pattern.py

```diff
@@ -112,9 +112,9 @@
                                              '33 FF ' + # xor edi, edi
                                              '48 8D 05 ? ? ? ? ' + # lea rax, ?
                                              '48 8b d9 ' + # mov rbx, rcx
-                                             '48 89 01 ', # mov [rcx], rax
+                                             '48 89 01 ' + # mov [rcx], rax
```

Comment:
I think this was in order to get it to pass with Retail builds. Does it find multiple occurrences on PTR?

Reply:
Yes, commenting the line under #todo found multiple occurences.

Reply:
A timeless classic… I guess the same as done with `DB2ConstructorLocation` is the way to go then :( Guess I'll do that the next time I trip over this when looking at a retail build. Luckily rarely do that.